arquillian/appclient/src/main/java/tck/arquillian/protocol/appclient/AppClientMethodExecutor.java

```diff
@@ -81,13 +83,13 @@ public TestResult invoke(TestMethodExecutor testMethodExecutor) {
         }
         String[] lines = appClient.readAll(config.getClientTimeout());
 
-        log.info(String.format("AppClient(%s) readAll returned %d lines\n", testMethod, lines.length));
+        log.fine(String.format("AppClient(%s) readAll returned %d lines\n", testMethod, lines.length));
         boolean sawStatus = false;
         MainStatus status = MainStatus.NOT_RUN;
         String reason = "None";
         String description = "None";
         for (String line : lines) {
-            System.out.println(line);
+            log.finer(line);
```

Comment:
This part could end up making it look a little odd if it's already a formatted log line. I also feel this should be at level `fine` not `finer`.